### ISSUE TITLE
Lighten the color of the punctuation in YAML files in dark mode

### ISSUE THREAD

**Issue by jkunimune** [typst/webapp-issues#355]
### Description

The brackets, colons, and commas are the same dark gray in dark mode as they are in light mode.  They should be changed to light gray to provide better color contrast.

![a screenshot of some YAML](https://github.com/typst/webapp-issues/assets/5607801/5a562034-d550-48ad-9df7-257f9c690cff)

### Use Case

The dark gray characters are almost impossible to see on a black background.  A different color would vastly improve readability.
